frontend/src/games/view/GamePage.tsx

```diff
@@ -49,6 +71,61 @@ const GamePage = () => {
         }
     }, [request, api, cohort, id]);
 
+    const onSave = (headers: GameHeader) => {
+        if (game === undefined) {
+            console.error('Game is unexpectedly undefined');
+            return;
+        }
+
+        updateRequest.onStart();
+
+        const chess = new Chess();
+        chess.loadPgn(game.pgn);
+        const headerMap = {
+            White: headers.white,
+            Black: headers.black,
+            Date: headers.date,
+            Result: headers.result,
+        };
+
+        for (const [name, value] of Object.entries(headerMap)) {
+            if (value) {
+                chess.setHeader(name, value);
+            }
+        }
+
+        const update: UpdateGameRequest = {
+            headers,
+            unlisted: true,
+            orientation: 'white',
```

Comment:
Maybe we should add the orientation to this first load popup?

Reply:
I thought similar.